python/common/business-registry-account/src/business_account/AccountService.py

```diff
@@ -0,0 +1,204 @@
+            return HTTPStatus.BAD_REQUEST
+        return HTTPStatus.OK
+
+    @classmethod
+    def update_entity(cls,
+                      business_registration: str,
+                      business_name: str,
+                      corp_type_code: str,
+                      state: str = None):
+        """Update an entity."""
+        auth_url = current_app.config.get('AUTH_SVC_URL')
+        account_svc_entity_url = f'{auth_url}/entities'
+
+        token = cls.get_bearer_token()
+
+        if not token:
+            return HTTPStatus.UNAUTHORIZED
+
+        # Create an entity record
+        entity_data = {
+            'businessIdentifier': business_registration,
+            'corpTypeCode': corp_type_code,
+            'name': business_name
+        }
+        if state:
+            entity_data['state'] = state
+
+        entity_record = requests.patch(
+            url=account_svc_entity_url + '/' + business_registration,
+            headers={**cls.CONTENT_TYPE_JSON,
+                     'Authorization': cls.BEARER + token},
+            data=json.dumps(entity_data),
+            timeout=cls.timeout
+        )
+
+        if entity_record.status_code != HTTPStatus.OK:
+            return HTTPStatus.BAD_REQUEST
+        return HTTPStatus.OK
+
+    @classmethod
+    def delete_affiliation(cls, account: int, business_registration: str) -> Dict:
+        """Affiliate a business to an account.
+
+        @TODO Update this when account affiliation is changed next sprint.
+        """
+        auth_url = current_app.config.get('AUTH_SVC_URL')
+        account_svc_entity_url = f'{auth_url}/entities'
+        account_svc_affiliate_url = f'{auth_url}/orgs/{account}/affiliations'
+
+        token = cls.get_bearer_token()
+
+        # Delete an account:business affiliation
+        affiliate = requests.delete(
+            url=account_svc_affiliate_url + '/' + business_registration,
+            headers={**cls.CONTENT_TYPE_JSON,
+                     'Authorization': cls.BEARER + token},
+            timeout=cls.timeout
+        )
+        # Delete an entity record
+        entity_record = requests.delete(
+            url=account_svc_entity_url + '/' + business_registration,
+            headers={**cls.CONTENT_TYPE_JSON,
+                     'Authorization': cls.BEARER + token},
+            timeout=cls.timeout
+        )
+
+        if affiliate.status_code != HTTPStatus.OK \
+                or entity_record.status_code not in (HTTPStatus.OK, HTTPStatus.NO_CONTENT):
+            return HTTPStatus.BAD_REQUEST
+        return HTTPStatus.OK
+
+    @classmethod
+    def get_account_by_affiliated_identifier(cls, identifier: str):
+        """Return the account affiliated to the business."""
+        token = cls.get_bearer_token()
+        auth_url = current_app.config.get('AUTH_SVC_URL')
+        url = f'{auth_url}/orgs?affiliation={identifier}'
+
+        res = requests.get(url,
```

Comment:
Same here https://github.com/bcgov/lear/blob/main/legal-api/src/legal_api/services/bootstrap.py#L284
May be change (flags) this to a parameter instead of global reference

Reply:
Updated and changed flags to be a parameter as per above